web/war/pom.xml

```diff
@@ -206,5 +206,230 @@
             </plugins>
             </build>
         </profile>
+        <profile>
+            <id>web-war-with-ui-plugins</id>
```

Comment:
Actually, strike that last change. Makes it obvious its for the war when building from the top level. Leave id as-is.